### PR TITLE
improve signed-compose job

### PR DESCRIPTION
**Pull request by sosiouxme**
* [ART-1127] give it an option to not add new builds.

### CODE REVIEW

File: jobs/build/signed-compose/build.groovy [hunk 1]
```diff
@@ -22,12 +22,21 @@ def initialize(advisory) {
...
+        return
+    }
     buildlib.elliott("${elliottOpts} change-state --state NEW_FILES ${advisoryOpt}")
 }
 
 // Search brew for, and then attach, any viable builds. Do this for
 // EL7 and EL8.
 def signedComposeAttachBuilds() {
+    if (!params.ATTACH_BUILDS) {
+        echo "Job configured not to attach builds; continuing using builds already attached"
```
Comment (tbielawa) [inline jobs/build/signed-compose/build.groovy]: tiny thing, but (almost) all the other `echo`s use `echo(msg)` (looks like I missed `Action may be required`).

File: jobs/build/signed-compose/build.groovy [hunk 2]
```diff
@@ -200,7 +220,7 @@ def signedComposeNewComposeEl8() {
     if ( params.DRY_RUN ) {
         currentBuild.description += "\nDry-run: EL8 Compose not actually built"
         echo("Packages which would have been added to rhaos-${params.BUILD_VERSION}-rhel-8-image-build tag:")
-        def testTagCmd = cmd + " --test"
+        def testTagCmd = tagCmd + " --test"
```
Comment (tbielawa) [inline jobs/build/signed-compose/build.groovy]: Was this busted before?

**Comment by tbielawa** [comment]
left comments, but overall
/lgtm

File: jobs/build/signed-compose/build.groovy (hunk 2 above)
Comment (sosiouxme) [reply jobs/build/signed-compose/build.groovy]: yes, but only if you did a dry run.

File: jobs/build/signed-compose/build.groovy (hunk 1 above)
Comment (sosiouxme) [reply jobs/build/signed-compose/build.groovy]: i don't mind changing it, but maybe we should establish a jenkins pipeline style guide :-P

**Pull request merged**